src/main/java/rotateit/service/github/label/OnLabelEventToViewListener.java

```diff
@@ -0,0 +1,13 @@
+package rotateit.service.github.label;
+
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import rotateit.domain.github.label.LabelEvent;
+
+@Component
```

Comment:
You've pushed just after my comments :) I would declare these in `ApplicationContext` and remove `@Component` for consistency